fe/fe-core/src/main/java/org/apache/doris/policy/Policy.java

```diff
@@ -18,79 +18,61 @@
 package org.apache.doris.policy;
 
 import org.apache.doris.analysis.CreatePolicyStmt;
-import org.apache.doris.analysis.Expr;
-import org.apache.doris.analysis.SqlParser;
-import org.apache.doris.analysis.SqlScanner;
 import org.apache.doris.analysis.UserIdentity;
 import org.apache.doris.catalog.Catalog;
 import org.apache.doris.catalog.Database;
 import org.apache.doris.catalog.Table;
 import org.apache.doris.common.AnalysisException;
 import org.apache.doris.common.io.Text;
 import org.apache.doris.common.io.Writable;
-import org.apache.doris.common.util.SqlParserUtils;
 import org.apache.doris.persist.gson.GsonPostProcessable;
 import org.apache.doris.persist.gson.GsonUtils;
 import org.apache.doris.qe.ConnectContext;
 
-import com.google.common.collect.Lists;
 import com.google.gson.annotations.SerializedName;
-import lombok.AllArgsConstructor;
 import lombok.Data;
+import org.apache.commons.lang.StringUtils;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 
-import java.io.DataInput;
 import java.io.DataOutput;
 import java.io.IOException;
-import java.io.StringReader;
 import java.util.List;
 
 /**
  * Save policy for filtering data.
  **/
 @Data
-@AllArgsConstructor
-public class Policy implements Writable, GsonPostProcessable {
-
-    public static final String ROW_POLICY = "ROW";
+public abstract class Policy implements Writable, GsonPostProcessable {
 
     private static final Logger LOG = LogManager.getLogger(Policy.class);
 
-    @SerializedName(value = "dbId")
-    private long dbId;
-
-    @SerializedName(value = "tableId")
-    private long tableId;
+    @SerializedName(value = "type")
+    protected PolicyTypeEnum type = null;
 
     @SerializedName(value = "policyName")
-    private String policyName;
-
-    /**
-     * ROW.
-     **/
-    @SerializedName(value = "type")
-    private PolicyTypeEnum type;
+    protected String policyName = null;
 
     /**
-     * PERMISSIVE | RESTRICTIVE, If multiple types exist, the last type prevails.
+     * Use for Serialization/deserialization.
      **/
-    @SerializedName(value = "filterType")
-    private final FilterType filterType;
-
-    private Expr wherePredicate;
+    @SerializedName(value = "originStmt")
+    protected String originStmt;
```

Comment:
Not all policy need to save origin stmt. I think we can move this field to TablePolicy.
For row policy, we save origin stmt only because we can not serialize the filter expr directly.
But for storage policy, there is only key value properties that can be serialized directly.